metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_v2.py

```diff
@@ -646,6 +646,12 @@ def gen_dataset_workunits(
             description=table.comment,
             qualifiedName=dataset_name,
             customProperties={**upstream_column_props},
+            externalUrl=self.get_external_url_for_table(
```

Comment:
For some companies, they might not use Snowsight. 

Can we put this feature behind a flag that is by default enabled?

extract_external_url?